CaseStudy.md

```diff
@@ -0,0 +1,166 @@
+
+
+<!-- You don't have to answer all the questions - just the ones relevant to your project. -->
+
+
+## Problem Statement
+The existing product has limitations making it challenging to scale the solution and quickly support new scenarios and customers.  
+
+![image](/docs/images/ProblemStatement_FishBone.jpg)
+
+<ins> Problem Areas Associated with the Broader Problem Statement: </ins>
+- <ins>A)	Model Generation:</ins> The extended turnaround time required for creation and updates of machine learning models hinders the ability to onboard new use cases or updating the existing ones. Absence of robust performance evaluation mechanisms for models adds complexity, making it difficult to assess and improve model effectiveness.
+- <ins>B)	Deployment Process:</ins> The current deployment process poses challenges, including semi-automated deployments with heavy Docker image sizes significantly slowing down the deployment process. Additionally, the tightly coupled model and application code hinders the agility required for swift updates and modifications.
+- <ins>C)	Deployment Pace:</ins> The inefficient deployment process leads to extended deployment timelines that can span several weeks. The lack of support for limited network bandwidth frequently increases these delays, causing deployments to become disproportionately time-consuming.
+- <ins>D)	Data Collection:</ins> Existing manual process for  data collection from the Edge devices is error prone and often leads to scalability issues.
+- <ins>E)	Cost:</ins> Higher network costs contributes to increased operational costs.
+  
+<ins> Few Stats on the Existing Architecture:</ins>
+
+|   Time taken for deploying multiple scenarios(5-10) against model      |     ~ Multiple Weeks                         |
+|------------------------------------------------------------------------|----------------------------------------------|
+| Model Package Container Size                                           | * 10GB for X86(AMD Build) <br/> * 7GB for ARM  
+
+
+
+<!-- As the libraries versions keep on changing, it is recommended to mention the version of library used in this project -->
+
+## Solution Overview
+To address these key challenges, the existing architecture has been revamped & reimagined by decoupling the model generation, from the business use case development and deployment process. This newer approach focuses on building a framework which can generate end-to-end machine learning use case with speed and agility.
+Technically, a ML use case can be defined as: A combination of single or multiple machine learning models coupled together by an orchestration mechanism to address a specific business problem.
+
+The conceptual design has 3 key components which function together to generate and deliver use cases to the target edge devices.
+
+![image](/docs/images/ConceptualDesign.jpg)
+
+- #### <ins>Machine Learning Model Factory:</ins>
```

Comment:
Can we add how many models we generated for the customer

Reply:
We generated one model for the customer. Shall we add that information?